### PR TITLE
[feat] add assistant metric

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- remove `description` field of ainft object.
- add `metric` field of assistant.

### CODE REVIEW

File: src/ai/assistant.ts [hunk 1]
```diff
@@ -435,4 +436,13 @@ export class Assistants extends FactoryBase {
       return 0;
     });
   }
+
+  private countThreads(items: any) {
+    if (typeof items === 'boolean' || !items) {
```
Comment (platfowner) [inline src/ai/assistant.ts:441]: question) how about something like `(typeof items !== 'object' || !items)` instead?

File: src/ai/assistant.ts [hunk 2]
```diff
@@ -435,4 +436,13 @@ export class Assistants extends FactoryBase {
...
+      return 0;
+    }
+    return _.sumBy(_.values(items), (item) => {
```
Comment (platfowner) [inline src/ai/assistant.ts:444]: minor) is lodash util dependency required? for example, can reduce() and Object.keys(), and Object.values() be used directly? ref: https://github.com/ainblockchain/ain-blockchain/blob/master/db/functions.js#L900

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: src/ai/assistant.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/ai/assistant.ts:441]: done!

File: src/ai/assistant.ts (hunk 2 above)
Comment (jiyoung-an) [reply src/ai/assistant.ts]: done!

**Comment by jiyoung-an** [comment]
Thank you for review @platfowner! now merging...

**Pull request merged**